futaba/sql/models/roles.py

```diff
@@ -169,11 +172,68 @@ def add_pingable_role_channel(self, guild, channel, role):
         assert guild == channel.guild
 
         ins = self.tb_pingable_role_channel.insert().values(
-            guild_id=guild.id, channel_id=channel.id, role_id=role.id
+            guild_id=guild.id, channel_id=channel.id, role_id=role.id, original_role_id=original.id
         )
 
         self.sql.execute(ins)
 
+    def get_pingable_role_from_original(self, guild, original_role):
+        sel = select(
+            [
+                self.tb_pingable_role_channel.c.role_id
+            ]
+        ).where(self.tb_pingable_role_channel.c.original_role_id == original_role.id)
+        result = self.sql.execute(sel).fetchall()
+        if len(result) == 0:
+            return None
+
+        # we get [0][0] because it's a tuple in a list and we only need the first one from each
+        return discord.utils.get(guild.roles, id=result[0][0])
+    
+    def get_original_from_pingable_role(self, role):
+        sel = select(
+            [
+                self.tb_pingable_role_channel.c.original_role_id
+            ]
+        ).where(self.tb_pingable_role_channel.c.role_id == role.id)
+        result = self.sql.execute(sel).fetchall()
+        if len(result) == 0:
+            return None
+        
+        return discord.utils.get(guild.roles, id=result[0][0])
+
+
+    def get_channels_from_role(self, guild, role):
+        if role == None:
```

Comment:
Minor, is preferred to do `role is None`.

Reply:
Done.